src/main/resources/entity-types/orders/simple_purchase_order_line.json5

```diff
@@ -0,0 +1,1625 @@
+      sourceAlias: 'pol',
+      dataType: {
+        dataType: 'arrayType',
+        itemDataType: {
+          dataType: 'objectType',
+          properties: [
+            {
+              name: 'claimed',
+              property: 'claimed',
+              dataType: {
+                dataType: 'booleanType',
+              },
+              queryable: false,
+              valueGetter: "( SELECT array_agg(elems.value->>'claimed') FROM jsonb_array_elements(:sourceAlias.jsonb->'claims') AS elems)",
+              filterValueGetter: "( SELECT array_agg(lower(elems.value->>'claimed')) FROM jsonb_array_elements(:sourceAlias.jsonb->'claims') AS elems)",
+              valueFunction: 'lower(:value)',
+              values: [
+                {
+                  value: 'true',
+                  label: 'True',
+                },
+                {
+                  value: 'false',
+                  label: 'False',
+                },
+              ],
+            },
+            {
+              name: 'sent',
+              property: 'sent',
+              dataType: {
+                dataType: 'dateType',
+              },
+              queryable: false,
+              valueGetter: "( SELECT array_agg(elems.value->>'sent') FROM jsonb_array_elements(:sourceAlias.jsonb->'claims') AS elems)",
+              filterValueGetter: "( SELECT array_agg(lower(elems.value->>'sent')) FROM jsonb_array_elements(:sourceAlias.jsonb->'claims') AS elems)",
+              valueFunction: 'lower(:value)',
+            },
+            {
+              name: 'grace',
+              property: 'grace',
+              dataType: {
+                dataType: 'integerType',
+              },
+              queryable: false,
+              valueGetter: "( SELECT array_agg(elems.value->>'grace') FROM jsonb_array_elements(:sourceAlias.jsonb->'claims') AS elems)",
+              filterValueGetter: "( SELECT array_agg(lower(elems.value->>'grace')) FROM jsonb_array_elements(:sourceAlias.jsonb->'claims') AS elems)",
+              valueFunction: 'lower(:value)',
+            },
+          ],
+        },
+      },
+      isIdColumn: false,
+      queryable: false,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'claims'",
+    },
+    {
+      name: 'claiming_active',
+      sourceAlias: 'pol',
+      dataType: {
+        dataType: 'booleanType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'claimingActive'",
+      values: [
+        {
+          value: 'true',
+          label: 'True',
+        },
+        {
+          value: 'false',
+          label: 'False',
+        },
+      ],
+    },
+    {
+      name: 'claiming_interval',
+      sourceAlias: 'pol',
+      dataType: {
+        dataType: 'integerType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'claimingInterval'",
```

Comment:
We should cast this to an integer and add a valueFunction that casts the value to an integer